CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
   Also add the image name (ref) of the image from "docker", with registry and tag.
   This is useful for traceability, when using `docker.io` or a tag like `latest`.
   Unfortunately the feature does not work with "docker-archive" or "docker-daemon".
+- Fix use of the image cache, when the home directory contains `@` characters.
```

Comment:
Thanks!  Please move the changelog entry to the 1.4.x section; this is a bugfix.

Reply:
I thought it was moved when cherry-picking to the release branch, but sure!